src/seedu/addressbook/data/person/Tagging.java

```diff
@@ -0,0 +1,45 @@
+package seedu.addressbook.data.person;
+
+import java.util.ArrayList;
+
+import seedu.addressbook.data.tag.Tag;
+
+public class Tagging {
+	private Tag tag;
+	private Person person;
+	private String tagModifier;
+	
+	private static ArrayList<Tagging> taggings;
```

Comment:
Tagging should be a simple association between `Person` and `Tag`.  The correct design should be `AddressBook` that holds the list of Taggings.